### PR TITLE
Update hide.js to show correct translation of Openmenu

### PR DESCRIPTION
**Pull request by try-and-succeed**
Currently, the text 'Openmenu' is shown instead of the real translation.
Therefore change the line 268 from ...Openmenu... to ...'+Openmenu+'...

### CODE REVIEW

**Comment by infograf768** [comment]
Nice find. PR works fine.
One more tester

**Comment by zero-24** [comment]
RTC based on testing. Thanks for the PR. @raabigramm <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7873">issues.joomla.org/joomla-cms/7873</a>.</sub>

File: templates/beez3/javascript/hide.js [hunk 1]
```diff
@@ -265,7 +265,7 @@ var mobileMenu = function(){
 		$menuWrapper = $('<div>', {id : 'menuwrapper', role: 'menubar'});
 
 		// create the menu opener and assign events
-		$mobile = $('<div>', {id: 'mobile_select'}).html('<h2><a href=#" id="menuopener" onclick="return false;"><span>Openmenu</span></a></h2>').show();
+		$mobile = $('<div>', {id: 'mobile_select'}).html('<h2><a href=#" id="menuopener" onclick="return false;"><span>'+Openmenu+'</span></a></h2>').show();
```
Comment (degobbis) [inline templates/beez3/javascript/hide.js]: On href I miss " like this href="#" not href=#"

Comment (zero-24) [inline templates/beez3/javascript/hide.js]: Can You open a PR to fix it. Else it get lost as you comment on an merged PR ;) Thanks

Comment (degobbis) [inline templates/beez3/javascript/hide.js]: Done -> https://github.com/joomla/joomla-cms/pull/8077

**Pull request merged**